packages/dashboard-frontend/src/services/factory-location-adapter/index.ts

```diff
@@ -58,7 +58,10 @@ export class FactoryLocationAdapter implements FactoryLocation {
   }
 
   public static isSshLocation(href: string): boolean {
-    return /^(ssh:\/\/)?git@[^:]+:.*\/[^/]+$/.test(href);
+    return (
+      !this.isHttpLocation(href) &&
+      /^(ssh:\/\/)?(?:([a-z_][a-z0-9_]{2,})@)?[^:]+:.*\/[^/]+$/.test(href)
```

Comment:
I don't see `git@` anymore

Reply:
The [origin issue](https://issues.redhat.com/browse/CRW-6566): 
```
Description of problem:
...
So, it is not possible to create a workspace from URLs like:
user1@repository.example.com:/home/user1/repositories/myrepo.git
When the first characters of the SSH URL are not git@, the button to create and open a workspace is disabled.
...
```

Reply:
We cowered these changes with tests.